classes/Profile.php

```diff
@@ -172,7 +172,11 @@ public static function getProfileAccesses($idProfile, $type = 'id_tab')
 				WHERE j.`id_profile` = '.(int) $idProfile);
 
                 foreach ($result as $row) {
-                    $idTab = self::findIdTabByAuthSlug($row['slug']);
+                    if ($type == 'class_name') {
+                        $idTab = self::findClassNameByAuthSlug($row['slug']);
+                    } else {
+                        $idTab = self::findIdTabByAuthSlug($row['slug']);
+                    }
 
                     self::$_cache_accesses[$idProfile][$type][$idTab][array_search('1', $row)] = '1';
```

Comment:
Can you please refacto this function in order to replace `$idTab` with `$tab`? We don't always deal with IDs so this name is now a non-sense.